bundles/org.openhab.binding.ferroamp/src/main/resources/OH-INF/thing/Channels.xml

```diff
@@ -0,0 +1,978 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<thing:thing-descriptions bindingId="ferroamp"
+	xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+	xmlns:thing="https://openhab.org/schemas/thing-description/v1.0.0"
+	xsi:schemaLocation="https://openhab.org/schemas/thing-description/v1.0.0 https://openhab.org/schemas/thing-description-1.0.0.xsd">
+	<channel-type id="ehub-wloadconsq-l1">
+		<item-type>Number:Energy</item-type>
+		<label>EHUB, WLoadConsq L1</label>
+		<category>Energy</category>
+		<state readOnly="true" pattern="%.2f %unit%"/>
+	</channel-type>
+	<channel-type id="ehub-wloadconsq-l2">
```

Comment:
You do not need to define multiple channel types for the same purpose just to have different labels and descriptions. Instead, you can override the label and description in the channels referencing the channel type.

There are also system channel types you could consider using - see:
https://github.com/openhab/openhab-core/blob/main/bundles/org.openhab.core.thing/src/main/java/org/openhab/core/thing/DefaultSystemChannelTypeProvider.java

Reply:
Done in channels.xml

Reply:
Please, see commit 20240607